packages/store-shared/package.json

```diff
@@ -9,15 +9,18 @@
   "exports": "./dist/index.js",
   "sideEffects": false,
   "scripts": {
-    "build": "tsc -b"
+    "build": "tsc -b",
+    "generateSchema": "typescript-json-schema src/validation/eventsValidator.ts Events -o src/validation/events.json --noExtraProps --required --ignoreErrors --strictNullChecks"
   },
   "dependencies": {
     "@withorbit/core": "0.0.1",
+    "ajv": "8.6.2",
```

Comment:
note: I had to manually specify this version (as opposed to `^8.6.2`) to avoid `store-fs` using the `ajv` hosted at the project root which is using version 6 of the package.